.github/workflows/static_checks.yml

```diff
@@ -10,6 +10,8 @@ jobs:
 
       - name: Install dependencies
         run: |
+          mkdir -p $HOME/.local/bin
```

Comment:
The `mkdir` is probably not needed, anything that installs to this folder will create it if missing.
It shouldn't be a problem if `PATH` points to a not-yet-created folder, it will just be skipped.